pages/1.12/cli/command-reference/dcos-security/dcos-security-cluster/dcos-security-cluster-oidc-modify/index.md

```diff
@@ -0,0 +1,32 @@
+---
+layout: layout.pug
+navigationTitle:  cos security cluster oidc modify
+title: cos security cluster oidc modify
+menuWeight: 60
+excerpt: Modifying an existing OIDC provider configuration
+enterprise: true
+---
+
+# Description
+
+The `cos security cluster oidc modify` command lets you modify an existing OIDC provider configuration. 
```

Comment:
`cos` -> `dcos` here and twice in the metadata above.

Reply:
Wow! Thanks for catching that!